### PR TITLE
The class calculates the area of the square

### PR DESCRIPTION
**Pull request by Sergiy-1998**
Define a class named `Shape` and its subclass `Square`.
The `Square` class has an init function which takes a length as argument.
Both classes have a `area` function which can print the area of the shape where `Shape`'s area is 0 by default. 

It might be used like this:
square = Square(3)
print(square.area())

### CODE REVIEW

**Comment by python-teacher** [comment]
Please change the name of this pull request.
It should be self-descriptive. It should be clear from the name what is this pull request about.

File: hw_9.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    def __init__(self, length):
+        self.length = length
+
+    def area(self):
+        return 0
+
+
+class Square(Shape):
+    def area(self):
+        return pow(self.length, 2)
```
Comment (python-teacher) [inline hw_9.py]: Why do you need this `pow` function here? It's better to simplify it.

**Review by python-teacher — CHANGES_REQUESTED** [review]

File: hw_9.py (hunk 1 above)
Comment (Sergiy-1998) [reply hw_9.py]: Done

File: hw_9.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    def __init__(self, length):
+        self.length = length
+
+    def area(self):
+        return 0
+
+
+class Square(Shape):
+    def area(self):
+        return self.length**2
```
Comment (python-teacher) [inline hw_9.py]: Is this formatting according to PEP-8?

Comment (Sergiy-1998) [reply hw_9.py]: I'm not sure about that

Comment (python-teacher) [reply hw_9.py]: You should check it when you're not sure about something.

Comment (Sergiy-1998) [reply hw_9.py]: I checked my option is correct

Comment (Sergiy-1998) [reply hw_9.py]: renamed file

**Review by python-teacher — APPROVED** [review]

**Comment by python-teacher** [comment]
:+1:

**Pull request merged**